Added conversations and messages to user via mailboxer gem.

Zawgor24 opened the pull request:
Used mailboxer gem.
Added conversation path.
Updated `message` button in user profile.
etc..

IlyaUmanets commented on app/controllers/conversations_controller.rb:
```diff
@@ -0,0 +1,36 @@
+  def create
+    recipient = User.find(params[:user_id])
+
+    receipt = current_user.send_message(recipient, 'Hi', recipient.email)
```
Is it a part of debugging ? Add this message to locale file if not.

IlyaUmanets commented on app/views/users/_message.html.slim:
```diff
@@ -0,0 +1,5 @@
+- conversation = current_user.mailbox.conversations.select { |con| con.recipients.include?(User.find_by(id: @user.id)) }
```
You mustn't pus this code here. Move it to helper or controller.

Zawgor24 committed:
Added direct messages to user and conversations.

Added direct messages to user and conversations.

Added direct messages to user and conversations.
